src/Knoop/Node/test/list.test.cpp

```diff
@@ -0,0 +1,123 @@
+#include "catch.hpp"
+#include "Knoop.hpp"
+
+using namespace njoy::Knoop;
+using Node_t = Node<int, std::string>;
+
+SCENARIO( "list" ){
+  // this lambda is a workaround for isuues in the interaction between
+  // C macros (like Catch's REQUIRE statement) and C++ template
+  // instatiations;
+  auto getInt =
+    [](auto&& node)
+    // auto&& respects and preserves const-ness
+    -> decltype(auto)
+    // decltype(auto) explicit return type preserves reference-ness
+    { return node.template get<int>(); };
+
+  auto getString =
+    [](auto&& node) -> decltype(auto)
+    { return node.template get<std::string>(); };
+
+  GIVEN( "a list Node" ){
+    WHEN( "values can be 'push_back'-ed" ){
```

Comment:
Is there a way to construct a list Node with a set of data instead of having to `push_back` for each one of them?

Can I do this? `auto aNode = Node_t{1, 2, 3, 4};`

Reply:
We could add this constructor. Good idea!

Note, you can currently do this with `push_back` in a single invocation.

```cpp
aNode.push_back(1, 2, "hello", 3, 4, "World");
```

Reply:
Done!